packages/config/config/devices/0x015f/mh-3901z.json

```diff
@@ -0,0 +1,129 @@
+{
+	"manufacturer": "McoHome Technology Co., Ltd.",
+	"manufacturerId": "0x015f",
+	"label": "MH3901-Z",
+	"description": "Boiler Thermostat",
+	"devices": [
+		{
+			"productType": "0x3910",
+			"productId": "0x5102"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "Temperature reporting",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 1,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Off",
+					"value": 0
+				},
+				{
+					"label": "Difference reporting only",
+					"value": 1
+				},
+				{
+					"label": "Interval reporting only",
+					"value": 2
+				},
+				{
+					"label": "Difference and interval reporting",
+					"value": 3
+				}
+			]
+		},
+		{
+			"#": "2",
+			"label": "Temperature difference setting",
+			"description": "Based on 0.1 °C unit: n * 0.1 °C = 0.5 °C",
+			"unit": "0.1 °C",
+			"valueSize": 2,
+			"minValue": 3,
+			"maxValue": 995,
+			"defaultValue": 5,
+			"unsigned": true
+		},
+		{
+			"#": "3",
+			"label": "Report interval time",
+			"description": "Based in 1s unit. Suggested > 30s",
+			"unit": "seconds",
+			"valueSize": 2,
+			"minValue": 10,
+			"maxValue": 32767,
+			"defaultValue": 60,
+			"unsigned": true
+		},
+		{
+			"#": "5",
+			"label": "Association 2 SET command retransmission times",
+			"description": "0: not resend, 1-127: resend times",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 127,
+			"defaultValue": 3
+		},
+		{
+			"#": "7",
+			"label": "Set the number of units for the heating period (menu P07)",
+			"description": "Total length of a heating cycle",
+			"valueSize": 1,
+			"minValue": 10,
+			"maxValue": 120,
+			"defaultValue": 20,
```

Comment:
What unit is this?

Reply:
This one is not easy to understand from the documentation. If I have to guess, it is called 'Energy Saving' mode in the documentation. See #4197 for documentation. 

It looks like a duty cycle, in which the length of the cycle, defined by parameter 7, and the length the heating is 'on' during this cycle is defined by parameter 8. If this is correct, the two values are describing a ratio and don't have a unit.

Reply:
Its minutes, same for the next param :)
![grafik](https://user-images.githubusercontent.com/17641229/153080375-f78e7bdb-e74d-42dc-8dd7-cd7257cb6adf.png)

Reply:
Ah, got it! Thanks!!